whois-api/src/main/java/net/ripe/db/whois/api/rest/WhoisSearchService.java

```diff
@@ -139,6 +137,7 @@ public Response search(
             @QueryParam("source") final Set<String> sources,
             @QueryParam("query-string") final String searchKey,
             @QueryParam("inverse-attribute") final Set<String> inverseAttributes,
+            @QueryParam("client-attribute") final String clientAttribute,
```

Comment:
Maybe rename to "client" to match QueryFlags (and it's not related to an RPSL attribute)

Reply:
Just having client is more confusing. MAy be client-flag?